src/module_PI/Raspberry_PI/main/PiApp.java

```diff
@@ -1,37 +1,46 @@
 package module_PI.Raspberry_PI.main;
 
+import java.io.*;
+import java.util.*;
 
 public class PiApp {
 
 	/**
 	 * @param args
 	 */
 	public static void main(String[] args) {
+		parseArgs(args);
 		SerialInputListener inputListener = new SerialInputListener();
 		Thread listener = new Thread(inputListener, "Listener");
 		listener.start();		
 		SerialOutput output = new SerialOutput();
+    	EmailNotifier notifier = new EmailNotifier("lastrellik@gmail.com", "Today's message");
+    	notifier.sendEmail();
 		
 		pause(10000);
-		output.sendTestComponentsPacket();
-		output.sendCalibratePacket();
 		output.sendArmPacket();
-		output.sendTriggerPacket();
-		output.sendSilencePacket();
-		
-		while(true){
-			output.sendArmPacket();
-			pause(5000);
-			output.sendTriggerPacket();
-			pause(2000);
-			output.sendSilencePacket();
-			pause(2000);
-			output.sendDisarmPacket();
-			pause(5000);
-			output.sendResetCalibrationPacket();
-			pause(5000);
-			output.sendCalibratePacket();
-			pause(5000);
+	}
+	
+	private static void parseArgs(String[] args){
+		int numOfArguments = 2;
+		if(args.length != numOfArguments) throw new IllegalArgumentException("Syntax is java PiApp <username> <password>");
```

Comment:
See my comment in loadGmailCredentials() in EmailNotifier.java. This is the line I was referring to that makes the writing to and then reading from a file not necessary / just extra work.